totalRP3/modules/dashboard/dashboard.lua

```diff
@@ -318,3 +294,343 @@ TRP3_API.events.listenToEvent(TRP3_API.events.WORKFLOW_ON_LOADED, function()
 		TRP3_API.toolbar.toolbarAddButton(Button_RPStatus);
 	end
 end);
+
+--- Mixin for dropdown menus on the dashboard status frame. These widgets
+--  come with an attached label for the control and a dropdown as a subframe.
+local DashboardStatusDropDownMixin = {};
```

Comment:
Could I suggest moving this to a new file? The dashboard file itself is quite big, it could use some splitting